### ISSUE TITLE
RCPT TO same address multiple times

### ISSUE THREAD

**Issue by zypA13510** [eleith/emailjs#242]
This happens when TO and CC contain the same address. Emailjs will try to RCPT TO that same address multiple times. And for me (using Exchange 365 SMTP server) it throws a timed out error.

Rfc2822 does not seem to forbid an address to appear in TO and CC at the same time. Also, this is possible using `mailx` (hostname, IP and mail address redacted):
```Console
$ echo "Testing email with the same TO and CC recipient." | mailx -v -r '"no-reply" <noreply@example.com>" -s "Test email" -S smtp="mail.example.com:25" -c Alice@example.com Alice@example.com
Resolving host mail.example.com . . . done.
Connecting to 10.0.0.1:25 . . . connected.
220 mail.example.com Microsoft ESMTP MAIL Service ready at Mon, 7 Oct 2019 12:46:57 +0800
>>> HELO localhost
250 mail.example.com Hello [10.0.0.1]
>>> MAIL FROM:<noreply@example.com>
250 2.1.0 Sender OK
>>> RCPT TO:<Alice@example.com>
250 2.1.5 Recipient OK
>>> DATA
354 Start mail input; end with <CRLF>.<CRLF>
>>> .
250 2.6.0 <●●●●●●●●●●●●●●●●●●●●●●●●●●noreply@example.com> [InternalId=●●●●●●●●] Queued mail for delivery
>>> QUIT
221 2.0.0 Service closing transmission channel
```
The email received has Alice on both TO and CC.

To fix this, I think a dedupe step for RCPT only should be added before SMTP send.

**Comment by zackschuster** [comment]
fixed in 265b88a04694a6cc2e3eb281e86a82193df7e9b8 :)

**Comment by zackschuster** [comment]
@zypA13510 this is now live in `3.0.0`.
